x-pack/plugins/maps/public/classes/sources/es_search_source/util/scaling_documenation_popover.tsx

```diff
@@ -0,0 +1,132 @@
+import { EuiButtonIcon, EuiPopover, EuiPopoverTitle, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  limitOptionLabel: string;
+  clustersOptionLabel: string;
+  maxResultWindow: string;
+  mvtOptionLabel: string;
+}
+
+interface State {
+  isPopoverOpen: boolean;
+}
+
+export class ScalingDocumenationPopover extends Component<Props, State> {
+  state: State = {
+    isPopoverOpen: false,
+  };
+
+  _togglePopover = () => {
+    this.setState((prevState) => ({
+      isPopoverOpen: !prevState.isPopoverOpen,
+    }));
+  };
+
+  _closePopover = () => {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  };
+
+  _renderContent() {
+    return (
+      <div>
+        <EuiPopoverTitle>
+          <FormattedMessage id="xpack.maps.scalingDocs.title" defaultMessage="Scaling" />
+        </EuiPopoverTitle>
+        <EuiText grow={false}>
+          <p>
+            <FormattedMessage
+              id="xpack.maps.scalingDocs.intro"
+              defaultMessage="Select the appropriate option for your use case."
+            />
+          </p>
+          <dl>
+            <dt>{this.props.limitOptionLabel}</dt>
+            <dd>
+              <p>
+                <FormattedMessage
+                  id="xpack.maps.scalingDocs.limitDetails"
+                  defaultMessage="Layer displays features from the first {maxResultWindow} documents. Results exceeding {maxResultWindow} are not displayed."
+                  values={{ maxResultWindow: this.props.maxResultWindow }}
+                />
+              </p>
+              <p>
+                <FormattedMessage
+                  id="xpack.maps.scalingDocs.limitUseCase"
+                  defaultMessage="Use this option to display medium data sets and create choropleth maps to compare statistics across boundaries."
+                />
+              </p>
+            </dd>
+
+            <dt>{this.props.clustersOptionLabel}</dt>
+            <dd>
+              <p>
+                <FormattedMessage
+                  id="xpack.maps.scalingDocs.clustersDetails"
+                  defaultMessage="When results exceed {maxResultWindow}, the layer uses GeoTile grid aggregation to group your documents into clusters and displays metrics for each cluster. When results are less then {maxResultWindow}, the layer displays features from individual documents."
+                  values={{ maxResultWindow: this.props.maxResultWindow }}
+                />
+              </p>
+              <p>
+                <FormattedMessage
+                  id="xpack.maps.scalingDocs.clustersUseCase"
+                  defaultMessage="Use this option to display large data sets. Does not support term joins."
+                />
+              </p>
+            </dd>
+
+            <dt>{this.props.mvtOptionLabel}</dt>
+            <dd>
+              <p>
+                <FormattedMessage
```

Comment:
Vector tiles doesn't include 10,000 in the title like the others.  Should it?

Reply:
no because it makes the name to long.